build.gradle

```diff
@@ -24,10 +24,10 @@ sourceCompatibility = '1.8'
 group = 'to.wetransform'
 
 project.ext {
-  haleVersion = '3.3.1'
+  haleVersion = '3.4.0-SNAPSHOT'
 }
 
-version = '3.3.1'
+version = '3.4.0-SNAPSHOT'
```

Comment:
Related to the comment above the version here should be `3.3.2-SNAPSHOT`.
If `3.4.0-SNAPSHOT` is used we also have a conflict with between the builds of master and devel branches - both will publish the same version of hale-cli:
https://builds.wetransform.to/job/hale/job/hale-cli~publish/
https://builds.wetransform.to/job/hale/job/hale-cli~publish(devel)/